src/services/metadata.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { PredefinedProperties } from '../utils/types';
+import { fetchAppsMetadata, MetadataJson } from './apps-metadata';
+
+export interface Metadata {
+    name: string;
+    url: string;
+    appColor: string;
+    hiddenInAppsMenu: boolean;
+    resources: unknown;
+}
+
+export interface StudyMetadata extends Metadata {
```

Comment:
Now we have StudyMetadata and MetadataStudy.
Maybe merge apps-metadata.ts and metadata.ts ?